src/tools/handlers/asset-handlers.ts

```
@@ -2,531 +2,531 @@ import { cleanObject } from '../../utils/safe-json.js';
+        ]);
+
+        const res = await executeAutomationRequest(tools, 'add_material_node', {
+          assetPath: params.assetPath,
+          nodeType: params.nodeType,
+          posX: params.posX,
+          posY: params.posY
         });
+        return ResponseFactory.success(res, 'Material node added successfully');
       }
+      default:
+        const res: any = await executeAutomationRequest(tools, action || 'manage_asset', args);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)